### ISSUE TITLE
tidyall cache files pollute the released tarballs

### ISSUE THREAD

**Issue by krop** [shlomif/fc-solve#99]
I didn't notice them before but they were already present in the 6.8 release and are still in the released tarball from https://fc-solve.shlomifish.org/download.html

```
# du -h freecell-solver-6.10.0/.tidyall.d 
0       freecell-solver-6.10.0/.tidyall.d/backups
56K     freecell-solver-6.10.0/.tidyall.d/cache/0
80K     freecell-solver-6.10.0/.tidyall.d/cache/b
72K     freecell-solver-6.10.0/.tidyall.d/cache/4
76K     freecell-solver-6.10.0/.tidyall.d/cache/3
72K     freecell-solver-6.10.0/.tidyall.d/cache/f
44K     freecell-solver-6.10.0/.tidyall.d/cache/c
52K     freecell-solver-6.10.0/.tidyall.d/cache/7
100K    freecell-solver-6.10.0/.tidyall.d/cache/a
72K     freecell-solver-6.10.0/.tidyall.d/cache/5
72K     freecell-solver-6.10.0/.tidyall.d/cache/d
44K     freecell-solver-6.10.0/.tidyall.d/cache/2
80K     freecell-solver-6.10.0/.tidyall.d/cache/1
80K     freecell-solver-6.10.0/.tidyall.d/cache/e
72K     freecell-solver-6.10.0/.tidyall.d/cache/9
88K     freecell-solver-6.10.0/.tidyall.d/cache/6
92K     freecell-solver-6.10.0/.tidyall.d/cache/8
1,2M    freecell-solver-6.10.0/.tidyall.d/cache
1,2M    freecell-solver-6.10.0/.tidyall.d
```

**Comment by shlomif** [comment]
Thanks for reporting this, @krop ! I agree this is an issue. I will try to fix it and mint a new release, soon. No commitment though.

**Comment by shlomif** [comment]
@krop : hi! I think I fixed it in 6.12.0 . Can I close the ticket now?
